config/default.yml

```
@@ -158,6 +158,11 @@ Minitest/GlobalExpectations:
   VersionAdded: '0.7'
   VersionChanged: '0.26'
 
+Minitest/HiddenTestMethod:
+  Description: 'Detects hidden (marked as `private` or `protected`) test methods.'
+  Enabled: pending
```

I think it's probably `Severity: warning`.

```suggestion
  Enabled: pending
  Severity: warning
```